templates/web/base/report/display_tools_mobile.html

```diff
@@ -0,0 +1,43 @@
+<div class="shadow-wrap">
```

Comment:
This template seems very similar to the non-mobile one, I can combine them later.